py-rse/project.Rmd

```diff
@@ -34,12 +34,13 @@ We'll be referencing the current files in our Zipf's Law project directory:
 │   ├── book_summary.sh
 │   ├── collate.py
 │   ├── countwords.py
-│   ├── mymodule.py
 │   ├── plotcounts.py
-│   └── rcparams.yml
+│   ├── plotparams.yml
+│   └── utilities.py
 ├── data
 │   ├── README.md
 │   ├── dracula.txt
+│   ├── risk.txt
```

Comment:
I made an earlier comment about the fact that we could simply create `data/risk.txt` on the fly in the correctness chapter (rather than have it randomly sitting in `data/` from the beginning), in which case it doesn't need to be in the project directory summary for any chapters that come before the correctness chapter.

Reply:
I agree that it shouldn't be in the list at the start of the chapter.